### DISCUSSION TITLE
End Frame Call fun is not working of cocostudio::timeline::ActionTimeline

### DISCUSSION THREAD

**Discussion by HassamChaudhary1** [axmolengine/axmol#1524] [General]
On **cocos-2dx-v4,** cocostudio::timeline::ActionTimeline* end frame callback is not working. It is missing callback.
Probably it is bcoz **cocos-2dx-v4** is supporting XStudio but I want to make it work with cocos studio.
Any idea how I can make it work with cocos studio same like **cocos-2dx-v3** version.?

```
cocostudio::timeline::ActionTimeline* timeLine;
timeLine->setLastFrameCallFunc([this]()
                                                      {
// Any process that will happen when received the last frame callback
                                                      });
```

**Comment by HassamChaudhary1** [comment]
Found the issue and fixed. Closing it.

**Reply by aismann** [reply]
Can we get more info about the fix too?
Maybe a PR is usefully?

**Reply by HassamChaudhary1** [reply]
I was using cocos studio and the existing animation wasn't working fine bcoz the animation frames were overlapping that was causing the issue on cocos-2dx-v4, So, I modified the animation and remove the animation overlapping, after that it's working fine.
**For Example:**
1st animation frames were : (0, 12)
2nd animation frames were : (10, 20)

**Modified To:**
1st animation frames are now : (0, 10)
2nd animation frames now : (10, 20)

It solved the issue.
